features/step_definitions/networking.rb

```diff
@@ -1480,6 +1480,9 @@
     @result = admin.cli_exec(:patch, resource: "network.operator", resource_name: "cluster", p: "{\"spec\":{\"defaultNetwork\":{\"ovnKubernetesConfig\":{\"gatewayConfig\":{\"routingViaHost\": true}}}}}", type: "merge")
   end
   raise "Failed to patch network operator for gateway mode" unless @result[:success]
+  # Sometimes CNO starts rolling out changes after few seconds have passed (gap). During that gap, if we check rollout status it would always be a pass from previous rollout result.Its not a bug.
+  # Keeping a 30 seconds harcoded time before we make following instruction (oc rollout status) worthy
+  step "30 seconds have passed"
```

Comment:
`oc patch` will cause creation of new pod and deletion of old pod.
So instead of hard wait (wait time too short can not guarantee the rollout, while wait time too long cost us extra time), we can get the pod's name before `oc patch`, and verify the old pod is gone after `oc patch`. Thus way, the wait time is dynamic and ensures the rollout.